auto_round/autoround.py

```diff
@@ -79,7 +86,7 @@ def __new__(
         # for adam
         enable_adam: bool = False,
         # for MLLM
-        extra_config: Union[ExtraConfig, list[ExtraConfig]] = None,
+        extra_config: Union[MLLMExtraConfig, list[MLLMExtraConfig]] = None,
```

Comment:
should we change it to a more specific name, as we have many extral configs like lr, enable_minmax_tuning,